src/pluginlaunch.c

```diff
@@ -402,7 +402,15 @@ plugin_launch (struct scan_globals *globals, struct scheduler_plugin *plugin,
   pluginlaunch_wait_for_free_process (main_kb);
   p = next_free_process (main_kb, plugin);
   if (p < 0)
-    return -1;
+    {
+      g_debug ("There are currently no free slot for running a new plugins, "
```

Comment:
Either `a new plugin` or `new plugins`. Maybe
```suggestion
      g_debug ("There are currently no free slot for starting a new plugin, "
```